src/dense.jl

```diff
@@ -48,12 +48,28 @@ end
 
 Base.length(dense::DenseOutput) = length(dense.opts.tout)
 
+tdir(ivp, solver::DenseOutput) = sign(solver.opts.tstop - ivp.t0)
+
 @compat function (::Type{DenseOutput{I}}){T,I}(ivp::IVP{T};
+                                               tstop = T(Inf),
+                                               tout::AbstractVector{T} = T[tstop],
                                                opts...)
+    if all(tout.>=ivp.t0)
+        tout = sort(tout)
```

Comment:
That's another way to handle it, but then which convention do we choose with `t0=0`?
- `tout=[0,-1/2,-1]`
- `tout=[-1,1/2,0]`